extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -162,13 +172,12 @@ export class AmpA4A extends AMP.BaseElement {
       this.win.ampA4aValidationKeys = this.getKeyInfoSets_();
     }
 
-    // TODO(tdrl): Temporary, while we're verifying whether this is an
-    // acceptable solution to the 'Safari on iOS doesn't fetch iframe src
-    // from cache' issue.  See https://github.com/ampproject/amphtml/issues/5614
     /** @private {?ArrayBuffer} */
     this.creativeBody_ = null;
-    /** @private {?string} */
-    this.experimentalNonAmpCreativeRenderMethod_ = null;
+
+    /** @type {!CROSS_ORIGIN_RENDERING_MODE} @visibleForTesting */
```

Comment:
Amazingly enough, yes.  I was worried about that, but it didn't complain.  Should it?